### PR TITLE
Projects hover

### PR DESCRIPTION
**Pull request by EgleJuske**
- hover effect on projects images

ADDITIONAL EDITS:
- deleted unnecessary gaps in `index.html`
- added `h2` and `p` main styles to `home-page.css` (deleted duplicating `h2` and `p` styles where I could find them)
- added `btn-hero` classes to `hero.css` so that buttons would be white

### CODE REVIEW

File: css/components/services.css [hunk 1]
```diff
@@ -7,9 +7,6 @@
     display: inline-block;
     width: 100%;
     text-align: center;
-    font-size: 25px;
-    font-weight: 700;
-    text-transform: capitalize;
     padding-bottom: 40px;
     padding-top: 40px;
```
Comment (Jjurgita) [inline css/components/services.css]: padding-top: 80px;

**Review by Jjurgita — CHANGES_REQUESTED** [review]
Very good :) only change h2 padding-top in services section

File: css/components/services.css (hunk 1 above)
Comment (EgleJuske) [reply css/components/services.css]: Thanks.
Changed the padding

**Review by Jjurgita — APPROVED** [review]

**Review by JurateJ — APPROVED** [review]

**Review by InetaVei — APPROVED** [review]
well done 👍

**Pull request merged**